### PR TITLE
Add a job posting sponsorship

### PR DESCRIPTION
**Pull request by ericholscher**
This is the start of a formal replacemnet for our job board.

It could still use some additional love in terms of sprucing up the content,
but wanted to get something basic that we can work with.

----
📚 Documentation preview 📚: https://writethedocs-www--2206.org.readthedocs.build/

### CODE REVIEW

**Comment by ericholscher** [comment]
@mxsasha @plaindocs Would love a review on this.

File: docs/sponsorship/jobs.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+This sponsorship includes: 
+
+* A job posting in our monthly newsletter jobs section (Over {{ newsletter_subs }} subscribers)
+* A highlighted job posting in our ``#job-posts-only`` Slack channel ({{ slack_members }} members)
+
+Sponsored entry format
+----------------------
+
+The main goal here is to provide the company information, and a link to the job posting.
+The format is:
```
Comment (plaindocs) [inline docs/sponsorship/jobs.rst]: This entry format is about putting your information front and centre, and putting the link to your job posting in front of our community.

File: docs/sponsorship/jobs.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+* Company name
+* Job title
+* Job Location / Remote
+* Salary (optional)
+* Link to job posting
+
+This will be used to create a newsletter entry, and a Slack post.
```
Comment (plaindocs) [inline docs/sponsorship/jobs.rst]:
```suggestion 
This information will be used to create a newsletter entry and a Slack post.
```

**Review by plaindocs — APPROVED** [review]
LGTM

**Review by mxsasha — APPROVED** [review]
Sounds great! Let's see how it works out, and then see if we need to iterate :)

File: docs/sponsorship/jobs.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+This sponsorship includes: 
+
+* A job posting in our monthly newsletter jobs section (Over {{ newsletter_subs }} subscribers)
+* A highlighted job posting in our ``#job-posts-only`` Slack channel ({{ slack_members }} members)
+
+Sponsored entry format
+----------------------
+
+The main goal here is to provide the company information, and a link to the job posting.
```
Comment (ericholscher) [inline docs/sponsorship/jobs.rst]:
```suggestion
This entry format is about putting your information front and centre, and putting the link to your job posting in front of our community.
```

**Pull request merged**